addon/components/time-series-chart.js

```diff
@@ -574,10 +573,10 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
       d3.select(element).classed('hovered', true);
 
       var time = data.labelTime != null ? data.labelTime : data.time;
```

Comment:
labelTime is a piece of Data that comes from the line series.  It's represented in a string on an attribute like so: 
``M0,3.989422804014327A3.989422804014327,3.989422804014327 0 1,1 0,-3.989422804014327A3.989422804014327,3.989422804014327 0 1,1 0,3.989422804014327Z``
That translates in the data property that is passed in.  Since that comes from the point on the SVG path from within the D3 code base I am unable to tweak that data.  Part of the problem comes from the density of the data too and the need for extra precision within D3.  Longer time periods can be off by 10+ seconds @_@